app/routes/ranger.$.tsx

```diff
@@ -4,29 +4,19 @@ import { redirect } from '@remix-run/node'
 export const loader: LoaderFunction = (context) => {
   handleRedirects(context)
 
-  return redirect('/ranger/v1')
+  return redirect('/ranger/latest')
 }
 
 function handleRedirects(context: Parameters<LoaderFunction>[0]) {
   const url = new URL(context.request.url)
-  // prettier-ignore
   const reactRangerV1List = [
-    {from: 'docs/overview',to: 'docs/guide/introduction',},
-    {from: 'docs/installation',to: 'docs/guide/installation',},
-    {from: 'examples/basic',to: 'docs/examples/react/basic',},
-    {from: 'examples/custom-steps',to: 'docs/examples/react/custom-steps',},
-    {from: 'examples/custom-styles',to: 'docs/examples/react/custom-styles',},
-    {from: 'examples/logarithmic-interpolator',to: 'docs/examples/react/logarithmic-interpolator',},
-    {from: 'examples/update-on-drag',to: 'docs/examples/react/update-on-drag',},
-    {from: '',to: '',},
+    { from: 'docs/overview', to: 'docs/introduction' },
+    { from: 'docs/installation', to: 'docs/installation' },
   ]
 
   reactRangerV1List.forEach((item) => {
     if (url.pathname.startsWith(`/ranger/v1/${item.from}`)) {
-      throw redirect(
-        `/ranger/v1/${item.to}?from=reactRangerV1&original=https://react-ranger-v1.tanstack.com/${item.from}`,
-        301
-      )
+      throw redirect(`/ranger/v0/${item.to}`)
     }
   })
```

Comment:
Hmmm, something is still wrong with this code, because `/ranger/v1/docs/overview` should redirect me to `/ranger/v0/docs/introduction`, but I get this screen instead:
<img width="1030" alt="image" src="https://github.com/TanStack/tanstack.com/assets/43729152/958be5c9-31c0-4138-a434-4e0ef6ec1a36">

Reply:
Yep, see the comment below